clipper/__init__.py

```diff
@@ -1,11 +1,11 @@
 import os
 
-def data_dir():
```

Comment:
@algaebrown It might be a good idea to keep current references in clipper, but provide the option for users to add new refs if they have them (eg. 

```
def data_dir(custom_refdir=None): 
  return os.path.join(os.path.dirname(__file__), 'data') if custom_refdir is None else custom_refdir
```
)